RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1439,9 +1599,17 @@ class DeepTauId : public deep_tau::DeepTauBase {
                                           electrons,
                                           muons,
                                           pfCands,
-                                          inner_grid,
+                                          *inner_grid_,
                                           tau_funcs,
                                           true);
+    checkInputs(
+        *eGammaTensor_[true], "input_inner_egamma", dnn_inputs_2017_v2::EgammaBlockInputs::NumberOfInputs, inner_grid_);
```

Comment:
```suggestion
    using dnn_inputs_2017_v2;
    checkInputs(
        *eGammaTensor_[true], "input_inner_egamma", EgammaBlockInputs::NumberOfInputs, inner_grid_);
```
can make this code a bit less verbose